### ISSUE TITLE
setup_windows.py was overwritten with linux version

### ISSUE THREAD

**Issue by gkarsai** [Freenove/Freenove_4WD_Smart_Car_Kit_for_Raspberry_Pi#55]
setup_windows.py now contains linux code (e.g. sudo....)  - it was overwritten in 2022?

**Comment by fire-eggs** [comment]
Yup, that's what it looks like to me. Overwritten on August 9, 2023.

Even the [previous version](https://github.com/Freenove/Freenove_4WD_Smart_Car_Kit_for_Raspberry_Pi/blob/cebe2360a725ed168f57a67921b63882f573982c/Code/setup_windows.py) is a little messed up, e.g. the message `Please run 'sudo python setup_windows.py' again`.

The previous version also is inconsistent with respect to the python version (a mix of `python` and `python3` commands).

Looking at the other changes in that August 9, 2023 commit, it appears the goal for setup files was to add additional messaging for error handling. So I'd try the previous version, keeping the above issues in mind.
